### ISSUE TITLE
Cats vs dogs

### ISSUE THREAD

**Issue by 072arushi** [gdsc-igdtuw22/ML-data-classification#2]

**Comment by mansi05041** [comment]
I would like to work on this issue

**Comment by 072arushi** [comment]
You have been assigned this issue

**Comment by Diya032** [comment]
@072arushi @Parul-Mann Pls, assign me this issue. Accuracy: 92% and validation accuracy: 81% obtained using the model. Satisfactory Classification is being done on new images.

**Comment by ASHWINISHARMABTCSAI118** [comment]
@072arushi With the help of neural network understanding the image as a 3-D matrix i.e., first red channel ,2nd green channel and thirdly blue channel with is nothing but rgb. Importing libraries like np , cv2, os ,random , matplotlib, I would like ti resolve the issue raised. Please kindly assign me with this !!

**Comment by ASHWINISHARMABTCSAI118** [comment]
@072arushi could you please assign this to me?

**Comment by Shreyg-27** [comment]
Hi @072arushi. Can this issue be assigned to me?
